tests/Integration/TransformationSystem/Test_Client_Transformation.py

```diff
@@ -1,25 +1,60 @@
 """ This is a test of the chain
     TransformationClient -> TransformationManagerHandler -> TransformationDB
 
-    It supposes that the DB is present, and that the service is running
+    It supposes that the DB is present, and that the TransformationManager service is running
+
+    Tests in TransformationClientChainID also require that the FileCatalogDB is present and that DataManagement/FileCatalog service is running
+    Finally the TSCatalog plugin must be configured in the Resources section and set in the Operatiions CatalogList, e.g.:
+
+    Operations
+    {
+        Services
+        {
+            Catalogs
+            {
+            CatalogList = DIRACFileCatalog, TSCatalog
+            DIRACFileCatalog
+            {
+            CatalogType = FileCatalog
+            AccessType = Read-Write
+            Status = Active
+            CatalogURL = DataManagement/FileCatalog
+            }
+            TSCatalog
+            {
+            CatalogType = TSCatalog
+            AccessType = Read-Write
+            Status = Active
+            CatalogURL = Transformation/TransformationManager
+            }
+
+        ...
```

Comment:
Originally this test was supposed to be for the "TransformationSystem only". With the test that you are adding here it supposes instead that also the DFC is up and running. I think you should move this new test in a separate module (this would also simplify the management of CI infrastructure)